packages/core/types/src/dml/index.ts

```diff
@@ -33,7 +28,7 @@ export interface IDmlEntity<
   Schema extends DMLSchema,
   Config extends IDmlEntityConfig
 > {
-  [IsDmlEntity]: true
+  __isDmlEntity: boolean
```

Comment:
We were using symbols since they don't leak in the autocomplete of TypeScript.

Reply:
I couldn't find any good solution to type it and use the symbol coming from elsewhere.
any suggestion?
we can't import the same symbol from `@medusajs/types`

Reply:
I can remove from typings, and just use the symbol here and potentially ignore TS if needed